Update the component doc template (#1073)
viamrobotics/docs: docs/components/component/model1.md

---
title: "Configure a Model1"
linkTitle: "Model1"
weight: 12
type: "docs"
description: "Configure a model1 arm."
tags: ["name", "components"]
draft: true
# SMEs:
---

The `model1` arm model supports the XYZ unit made by Some Company(INSERT LINK AS APPLICABLE).
Optional additional description/information.

{{< tabs >}}
{{% tab name="Config Builder" %}}

On the **COMPONENTS** sub-tab, navigate to the **Create Component** menu.
Enter a name for your arm, select the `arm` type, and select the `model1` model.

![Creation of a `model1` arm in the Viam app config builder.](../img/model1-builder.png)

> {{% /tab %}}

Reviewer: ```suggestion
Click **Create Component** and then fill in the attributes for your model.

{{% /tab %}}
```

Author: done (with lowercase c)

---
title: "Configure a Model1"
linkTitle: "Model1"
weight: 12
type: "docs"
description: "Configure a model1 arm."
tags: ["name", "components"]
draft: true
# SMEs:
---

The `model1` arm model supports the XYZ unit made by Some Company(INSERT LINK AS APPLICABLE).
Optional additional description/information.

{{< tabs >}}
{{% tab name="Config Builder" %}}

> On the **COMPONENTS** sub-tab, navigate to the **Create Component** menu.

Author: good call

---
title: "Configure a Model1"
linkTitle: "Model1"
weight: 12
type: "docs"
description: "Configure a model1 arm."
tags: ["name", "components"]
draft: true
# SMEs:
---

The `model1` arm model supports the XYZ unit made by Some Company(INSERT LINK AS APPLICABLE).
Optional additional description/information.

{{< tabs >}}
{{% tab name="Config Builder" %}}

Navigate to the **config** tab of your robot's page in [the Viam app](https://app.viam.com).

> Click on the **Components** sub-tab and navigate to the **Create component** menu.

Author: ```suggestion
Click on the **Components** subtab and navigate to the **Create component** menu.

```

Enter a name for your arm, select the `arm` type, and select the `model1` model.

![Creation of a `model1` arm in the Viam app config builder.](../img/model1-builder.png)

> Click **Create Component** and then fill in the attributes for your model.

Author: ```suggestion
Click **Create component** and then fill in the attributes for your model.
```
